naxsi_src/naxsi_utils.c

```diff
@@ -648,7 +648,11 @@ ngx_http_wlr_finalize_hashtables(ngx_conf_t *cf, ngx_http_dummy_loc_conf_t  *dlc
   }
 
   FILE *file;
-  char *fname = "/etc/nginx/whitelist.txt";
+  char fname[100];
```

Comment:
Please use 1024 instead of 100 because the max path size on *nix machines is 1024

Reply:
Done